src/Demo.tsx

```diff
@@ -14,9 +14,8 @@ export interface DemoState {
   infoText: string;
 }
 
-export default class Demo extends React.Component<any, DemoState> {
+export default class Demo extends React.Component<{}, DemoState> {
   state: DemoState;
```

Comment:
You should be able to remove this line too.

Reply:
Missed this. Updated.